src/drag-and-drop.js

```diff
@@ -0,0 +1,28 @@
+let dragged;
+
+function allowDrop(ev) {
+  ev.preventDefault();
+}
+
+function drag(ev) {
+  dragged = this;
+  ev.dataTransfer.setData('text', this.outerHTML);
+}
+
+function drop(ev) {
+  if (dragged !== this) {
+    this.parentNode.removeChild(dragged);
+    const task = ev.dataTransfer.getData('text');
+    this.insertAdjacentHTML('beforebegin', task);
+    const dropped = this.previousSibling;
+    dropped.addEventListener('dragstart', drag, false);
+    dropped.addEventListener('dragover', allowDrop, false);
+    dropped.addEventListener('drop', drop, false);
```

Comment:
* I suggest you create a default list of Todos so that these EventListeners can be carried out on